src/declarativewebutils.cpp

```diff
@@ -24,9 +24,12 @@
 #include "declarativewebutils.h"
 #include "qmozcontext.h"
 
+static QString userOpenSearchPath = QDir::homePath() + "/.local/share/org.sailfishos/sailfish-browser/opensearch/";
 static const QString gSystemComponentsTimeStamp("/var/lib/_MOZEMBED_CACHE_CLEAN_");
 static const QString gProfilePath("/.mozilla/mozembed");
-static const QString gOpenSearchPath("/usr/lib/mozembedlite/chrome/embedlite/content/");
+static const QStringList gOpenSearchPathList = QStringList() \
+                                                << QString("/usr/lib/mozembedlite/chrome/embedlite/content/") \
+                                                << userOpenSearchPath;
```

Comment:
Related to the first comment would be nice if we'd encapsulate search engines to a class and this way share more code between webutils and settings. Static methods for paths and for parsing xmls should be enough.

You could add search.pri under "src" directory and include it in both src.pro and settings.pro. The search.pri could then include all needed header and source files.